docs/adr/005-administrative-user-interface.md

```diff
@@ -0,0 +1,29 @@
+# Administrative user interface
+
+Date: 20-09-2023
+
+## Status
+
+Proposed
+
+## Context
+
+We need to create an administrative interface that is accessible through a browser and that we can build quickly. The
+first plan about the administrative interface was to utilize the API and make it independent of the database.
+
+## Decision
+
+It has been decided to switch the administrative interface to [Easy admin](https://github.com/EasyCorp/EasyAdminBundle)
+bundle, which utilize the database and the entities to generate an administrative user interface.
+
```

Comment:
I think we should add: This is also within the lines of the first part of this project, wich is to upgrade the existing Evendatabase where we also use Easy Admin. Right now there is no reason why Easy admin should not be a sufficient solution to editors in the future.

Reply:
Added